hash_commands.go

```diff
@@ -388,6 +394,7 @@ func (c cmdable) HPExpireAtWithArgs(ctx context.Context, key string, tm time.Tim
 // HPersist - Removes the expiration time from specified fields in a hash.
 // Accepts a key and the fields themselves.
 // This command ensures that each field specified will have its expiration removed if present.
+// Available since Redis 7.4 CE.
 // For more information - https://redis.io/commands/hpersist/
```

Comment:
```suggestion
// HPersist - Removes the expiration time from specified fields in a hash.
// Accepts a key and the fields themselves.
// This command ensures that each field specified will have its expiration removed if present.
// Available since Redis 7.4 CE.
// For more information refer to [HPersist Documentation].
//
// [HPEXPIRE Documentation]: https://redis.io/commands/hpersist/
```